EIPS/eip-5058.md

```diff
@@ -0,0 +1,205 @@
+---
+eip: 5058
+title: Lockable ERC-721 Standard
```

Comment:
Can we do extend this to ERC-1155?

Reply:
Yes, it can be extended to ERC-1155，whether it needs to be implemented in this proposal ?

Reply:
Yeah, I think it would be better if we implement both ERC-721 and ERC-1155 at once. If it makes complicated, just ignore this comment. Thanks.

Reply:
> Yeah, I think it would be better if we implement both ERC-721 and ERC-1155 at once. If it makes complicated, just ignore this comment. Thanks.

ERC1155 has few locked usage scenarios. The current protocol is mainly for ERC721, and ERC1155 will be extended later.

Reply:
> Yeah, I think it would be better if we implement both ERC-721 and ERC-1155 at once. If it makes complicated, just ignore this comment. Thanks.

ERC1155 has few locked usage scenarios. The current protocol is mainly for ERC721.